tests/test_cluster_persistence.py

```
@@ -0,0 +1,637 @@
+        assert mock_client.post.call_count == 1
+
+    async def test_records_failure_on_error(self):
+        ft = FailureTracker(failure_threshold=5)
+        mgr = ClusterManager(failure_tracker=ft)
+        await mgr.register_worker(_make_worker("w1", capabilities=["chat"], load=0.1, url="http://w1:8000"))
+
+        mock_client = AsyncMock(spec=httpx.AsyncClient)
+        mock_client.post.side_effect = Exception("connection refused")
+
+        router = TaskRouter(mgr, mock_client)
+        data, name = await router.route_request("chat", "POST", "/v1/chat/completions", {})
```

Ruff (0.16.0)

[warning] Unpacked variable `name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)